### DISCUSSION TITLE
Shared Element transition does not work after adding @react-native-firebase on iOS

### DISCUSSION THREAD

**Discussion by esh-g** [software-mansion/react-native-reanimated#4691] [Q&A] [answered]
I'm using expo with `reanimated@3.3.0` and shared element transitions were working fine on iOS, but then I added Firebase(@react-native-firebase) and it broke the Shared transitions...

**Comment by tjzel** [comment, marked as answer]
@esh-g Please open an issue request with repro so we can work on that.

**Reply by esh-g** [reply]
Okay, I'll do that!

**Reply by Darren120** [reply]
WHere is the repo?!??

**Comment by incode-mike-bytenko** [comment]
Same problem. To repro just setup basic project with transition (like in reanimated Examples) and after that setup firebase via their instructions.

**Reply by incode-mike-bytenko** [reply]
https://github.com/software-mansion/react-native-reanimated/issues/4966
